src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/adjourncase/AdjournCaseAboutToSubmitHandler.java

```diff
@@ -141,4 +147,19 @@ private static HearingChannel getNextHearingChannel(SscsCaseData caseData) {
                         hearingChannel.getValueTribunals()))
                 .findFirst().orElse(HearingChannel.PAPER);
     }
+
+    private static void updateExcludedPanelMembers(SscsCaseData caseData) {
+        Adjournment adjournment = caseData.getAdjournment();
+        AdjournCasePanelMembersExcluded panelMemberExcluded = adjournment.getPanelMembersExcluded();
```

Comment:
Minor - rename variable to panelMember**s**Excluded

Reply:
done